src/main/java/seedu/taskboss/model/ModelManager.java

```diff
@@ -76,6 +76,11 @@ public void undoTaskboss() throws EmptyStackException {
         indicateTaskBossChanged();
     }
 
+    @Override
+    public void saveTaskboss() {
+        indicateTaskBossChanged();
```

Comment:
saveTaskboss() is not exactly doing the saving here, so I think we can change the method name to something more focused like indicateTaskBossChangedToSave(); or anything that is more concise :)